### DISCUSSION TITLE
Is posible use Autogluon to solve a multi label image clasification problem?

### DISCUSSION THREAD

**Discussion by msanov** [autogluon/autogluon#2396] [Q&A]
In the documentation i only see datasets with one label per image, its possible use Autogluon in images with multiple labels?

**Comment by rxjx** [comment]
https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-multilabel.html

**Reply by msanov** [reply]
I'm asking about image problems, no tabular data.

**Reply by rxjx** [reply]
Ah sorry, my bad. But perhaps you could use a similar approach, i.e. one classifier per label with a possible conditional dependence on prior labels? Although ideally you could also create a custom network with the appropriate final layer.

**Comment by mdabros** [comment]
I am looking for the same functionality but for multi target/label image regression. Using pytorch, for instance, it is quite easy to define a neural network with the required number of outputs. It is also much more efficient than having a separate regressor/classifier per output. So my follow up question is if it is possible to define a custom number of outputs for image regression/classification problems in autogluon? Would be nice to be able to utilize the nice features of autogluon for these problem types.
